likecoin/chain.json

```diff
@@ -24,12 +24,16 @@
     },
     "codebase": {
         "git_repo": "https://github.com/likecoin/likecoin-chain",
-        "recommended_version": "v2.0.0",
+        "recommended_version": "v2.0.2",
         "compatible_versions": [
-            "v2.0.0"
+            "v2.0.2"
```

Comment:
Thanks for the update. Is v2.0.0 really not compatible with v2.0.2?

Reply:
2.0.0 would panic when processing a specific block related to a unbonding validator pre-prefix change.
https://blog.like.co/en/2022-05-27-chain-halt-incident-report/
https://github.com/cosmos/cosmos-sdk/pull/12303

It is "compatible" in a sense that if you skip the problematic block through snapshot/statesync it would work, but I don't think we should list it here for safety

Reply:
OK thanks for confirming